src/Grid/grid.jl

```diff
@@ -392,8 +397,13 @@ Returns an iterateable face skeleton. The skeleton consists of `FaceIndex` that
 """
 faceskeleton(top::ExclusiveTopology, grid::AbstractGrid) =  top.face_skeleton
 
-toglobal(grid::Grid,vertexidx::VertexIndex) = vertices(getcells(grid,vertexidx[1]))[vertexidx[2]]
-toglobal(grid::Grid,vertexidx::Vector{VertexIndex}) = unique(toglobal.((grid,),vertexidx))
+"""
+    toglobal(grid::AbstractGrid, vertexidx::VertexIndex)
+    toglobal(grid::AbstractGrid, vertexidx::Vector{VertexIndex}) -> Vector{Int}
+This function takes the local vertex representation (a `VertexIndex`) and looks up the unique global id (an `Int`).
```

Comment:
See above.

Reply:
> Also I notices two undocumented convenience functions https://github.com/Ferrite-FEM/Ferrite.jl/blob/master/src/Grid/grid.jl#L395-L396 , maybe we could add these to the docs or are they for purely internal use?

https://github.com/Ferrite-FEM/Ferrite.jl/issues/453

Reply:
Since Mischa will use it as well in his yet to be published documentation example, it is not internal per se